src/audio/mixer.c

```diff
@@ -560,6 +562,16 @@ static int mixout_params(struct comp_dev *dev,
 	params->buffer_fmt = md->base_cfg.audio_fmt.interleaving_style;
 	params->buffer.size = md->base_cfg.ibs;
 
+	/* update each sink format */
+	sink = list_first_item(&dev->bsink_list, struct comp_buffer, source_list);
+	sink->stream.channels = md->base_cfg.audio_fmt.channels_count;
+	sink->stream.rate = md->base_cfg.audio_fmt.sampling_frequency;
+	sink->stream.frame_fmt	= (md->base_cfg.audio_fmt.valid_bit_depth >> 3) - 2;
+	sink->buffer_fmt = md->base_cfg.audio_fmt.interleaving_style;
+
+	for (i = 0; i < SOF_IPC_MAX_CHANNELS; i++)
```

Comment:
can we comment what we are doing with the channel map here ?

Reply:
sure